src/server/lib/middleware/abTestDemo.ts

```diff
@@ -1,64 +0,0 @@
-import { ResponseWithServerStateLocals } from '@/server/models/Express';
```

Comment:
this isn't being used, as it's an example file on how to possibly run an test for a given route, so we may still want to keep this, or move it into the docs

Reply:
Good shout, I've moved this code to the A/B Test docs